add-ons/debezium/README.md

```diff
@@ -0,0 +1,99 @@
+# Deploying Debezium on OpenShift
+
+This addon provides an easy way to install [Debezium](http://debezium.io/) on [Minishift](https://github.com/minishift/minishift).
+
+Prerequisites:
+
+ * [Installed and running](https://docs.openshift.org/latest/minishift/getting-started/installing.html) Minishift
+ * [Installed](https://docs.openshift.org/latest/minishift/command-ref/minishift_oc-env.html) OpenShift CLI
+
+## Start Minishift
+The deployment works with Minishift default resources - 2 CPUs and 2 GB RAM. Nevertheless we prefer to allocate more memory (4 GB) available to Debezium and associated infrastructure. Start Minishift using following command to provide more RAM.
+```
+$ minishift start --memory=4GB
+```
+
+## Add-on installation
+Clone this repository onto your local machine and then install the add-on via:
+```
+    $ minishift addons install <path_to_directory_containing_this_readme>
+```
+
+## Debezium start
+Deploy the Kafka broker, Kafka Connect with Debezium and MySQL connector
+
+```
+minishift addon apply -a DEBEZIUM_VERSION=0.6.1 -a DEBEZIUM_PLUGIN=mysql -a PROJECT=myproject debezium
+```
+
+After a while all parts should be up and running:
+
+```
+oc get pods
+NAME                    READY     STATUS      RESTARTS   AGE
+debezium-1-build        0/1       Completed   0          3m
+debezium-2-build        0/1       Completed   0          3m
+kafka-0                 1/1       Running     2          3m
+kafka-1                 1/1       Running     0          2m
+kafka-2                 1/1       Running     0          2m
+kafka-connect-3-3v4n9   1/1       Running     1          3m
+zookeeper-0             1/1       Running     0          3m
+```
```

Comment:
I now see one `debezium-1-build  `, I think this is intentional, please update the readme for the same. 
```
$ oc get pods
NAME                    READY     STATUS      RESTARTS   AGE
debezium-1-build        0/1       Completed   0          7m
kafka-0                 1/1       Running     1          7m
kafka-1                 1/1       Running     0          7m
kafka-2                 1/1       Running     0          6m
kafka-connect-2-rddx7   1/1       Running     2          6m
zookeeper-0             1/1       Running     0          7m
```

Reply:
Well if there is only one build pod or two is just an issue of timing - whether the second build is scheduled before the first starts. I most of the time get two pod.